src/web/forms/fields/block/Block.js

```diff
@@ -52,12 +52,14 @@ export const BlockBody = ({
 export const Block = ({
   label,
   children,
+  alerts,
```

Comment:
Why did you add `alerts` prop to `Block`?

Reply:
Fixed thanks to linter &